CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## Version 0.2.1: June 2, 2025
+
+* Support commands in config [#796](https://github.com/clangd/vscode-clangd/pull/796) [#48](https://github.com/clangd/vscode-clangd/issues/48)
```

Comment:
If you're going to do a new release as part of this PR, please list other new changes since the last release:

 * #822
 * #837
 * #848

Reply:
Yes, good catch. Still getting the hang of the new version numbering system.

Reply:
Works for me.

Reply:
Created #856 for the new pre-release.